src/man/sssd-ipa.5.xml

```diff
@@ -112,7 +112,8 @@
                         <para>
                             Optional. May be set on machines where the
                             hostname(5) does not reflect the fully qualified
-                            name used in the IPA domain to identify this host.
+			    name used in the IPA domain to identify this host.
```

Comment:
Please, mind the indentation here.

I'd also split this in 2 patches.
1st patch: MAN: Improve ipa_hostname description
2nd patch: IPA: check if IPA hostname is fully qualified.